proto/ibc/applications/transfer/v1/query.proto

```diff
@@ -30,6 +30,11 @@ service Query {
   rpc DenomHash(QueryDenomHashRequest) returns (QueryDenomHashResponse) {
     option (google.api.http).get = "/ibc/apps/transfer/v1/denom_hashes/{trace}";
   }
+
+  // EscrowAddress returns the escrow address for a particular port and channel id.
+  rpc EscrowAddress(QueryEscrowAddressRequest) returns (QueryEscrowAddressResponse) {
+    option (google.api.http).get = "/ibc/apps/transfer/v1/escrow_address/port/{port_id}/channel/{channel_id}";
```

Comment:
Sorry, @chatton, I just realized that maybe the REST url should be different (hence my comment on Slack as well)...

```suggestion
    option (google.api.http).get = "/ibc/apps/transfer/v1/channels/{channel_id}ports/{port_id}/escrow_address";
```